float64 fpos_x
float64 fpos_y
float64 fpos_z
---
bool task
